Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js

```diff
@@ -1,16 +1,88 @@
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
+import {
+  Button,
+  TextField,
+  Typography,
+  Container,
+  Box,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import Introduction from '../Introduction';
 import Title from '../Title';
+import Copyright from '../components/Copyright';
```

Comment:
Is it not import Copyright from '../Copyright';

Reply:
Changed